crates/typst-syntax/src/ast.rs

```diff
@@ -1563,6 +1563,11 @@ node! {
 }
 
 impl<'a> FuncCall<'a> {
+    /// The function identifier.
+    pub fn ident(self) -> Ident<'a> {
```

Comment:
The callee may not be an identifier, so this operation is a bit ill-defined. You should match on `callee` instead.

Reply:
Thanks for the review.

I see `#"A"("ABC")` can be parsed to `FuncCall` and it has no identifier, so this operation is not good indeed. To make it better, I turned to let this function return `Option<Ident<'a>>` instead just now.

Reply:
why not just `match call.callee() { ast::Expr::Ident(ident) => .., _ => .. }` ?

Reply:
Indeed. I've modified the code according to your suggestion.